tests/smoke-tests/--typescript.test.ts

```diff
@@ -86,6 +86,18 @@ for (let packageManager of SUPPORTED_PACKAGE_MANAGERS) {
         'template-registry.js.map',
       ]);
 
+      expect(distContents.hashed.length).toBe(4);
```

Comment:
Why is it 4, and is this necessarily the case? Or are we assuming rollup implementation details here, and potentially making the test brittle?

Reply:
rollupSomething js + map
template-only js + map

Previously the rollup files were filtered out

I do not believe the test would become brittle